docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -573,6 +581,132 @@ To receive the reply of processing command on the server side:
 > This feature supports Tizen 4.0 and Higher for Mobile.
 
 
+## Sending and Processing a Search Command
+
+To send a search command to the client from the server side:
+
+1. Create the media controller client handle using the `mc_client_create()`:
+
+   ```
+   ret = mc_client_create(&g_client_h);
+   ```
+
+2. Retrieve the server name using the `mc_client_foreach_server()`:
+
+   ```
+   bool
+   activated_server_cb(const char *server_name, void *user_data)
+   {
+     GList **server_list = (GList **)user_data;
+
+     if (!server_list || !server_name) return FALSE;
+     *server_list = g_list_append(*client_list, g_strdup(server_name));
+
+     return TRUE;
+   }
+
+   GList *server_list = NULL;
+
+   ret = mc_client_foreach_server(g_client_h, activated_server_cb, &server_list);
+   ```
+
+3. Create the search handle using the `mc_search_create()`:
+
+   ```
+   ret = mc_search_create(&g_search_h);
+   ```
+
+4. Set the condition with [a content type](#media-controller-content-type-attributes), [a search category](#media-controller-search-category-attributes) and a search keyword using the `mc_search_set_condition()`:
+
+   ```
+   ret = mc_search_set_condition(g_search_h, MC_CONTENT_TYPE_MUSIC, MC_SEARCH_TITLE, "keyword", NULL);
+   ```
+
+5. Send the search command to the server using the corresponding `mc_client_send_search_cmd()`. Use the server name retrieved in the previous step to identify the server.
+For example, to send a your own event to the client, use the `mc_client_send_search_cmd()` with the search handle in the third parameter:
```

Comment:
Please rephrase it as:

For example, to send your own event to the client, use the `mc_client_send_search_cmd()` with the search handle in the third parameter: